requirements_dev.txt

```diff
@@ -8,6 +8,7 @@ flake8==3.7.5
 mock==2.0.0
 pre-commit==1.14.3
 pytest==4.2.0
+pytest-cov==2.6.1
```

Comment:
I've had trouble in the past with `pytest-cov` not initializing coverage early enough, so I prefer using `coverage run path/to/pytest`.

`make coverage` _should_ do the right thing, but I think it needs a fix to work when using pyenv.